### DISCUSSION TITLE
Generate unused strings

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#8745] [Q&A]
Hi there ^^ !

For PR #8558, I need to retrieve a translation according to a key that is in a list.

But as far as I know, it's impossible to generate keys (strings.txt) that aren't explicitly used. Is there a way around this?

Thanks in advance ^^ 

![image](https://github.com/user-attachments/assets/8b2b0802-06ca-4dd9-866a-25f502810df9)

**Comment by biodranik** [comment]
Can you please clarify your use-case? Why can't you list only supported values?

**Reply by ghost** [reply]
I list supported value. But to not use a switch to add text on radio button, I prefere to get translated string depend on "value".

Then I saw that for wheelchair accessibility, for example, type strings were used. It's an ugly way to use it (for my code), I admit, but I can't think of any simpler way.

Do you have any ideas on how to make it cleaner?

![image](https://github.com/user-attachments/assets/4b409a12-c21e-4c49-8b9f-faa561b90834)
![Screenshot_20240721_165336](https://github.com/user-attachments/assets/21ab8302-c157-4164-b518-d0003e266e67)

**Reply by biodranik** [reply]
1. I do not understand what is "ugly" in translating 4 strings.
2. Translations should be very clear for your grandma. Will she understand what "Partially" or "Only" means?

**Reply by ghost** [reply]
That's not translating 4 strings, that's generate them for strings.txt file, because they are not used specifically in the code, it hangs and the generation script returns an error.
